### ISSUE TITLE
Documentation isn't aligned with what worked for me. Not sure where the error is.

### ISSUE THREAD

**Issue by pablobarria** [webpack-contrib/postcss-loader#481]
I just updated dependencies on a project from postcss-loader 3.0.0 to 4.0.2

My previous webpack config file, using *postcss-present-env* and *postcss-nested* looked like this:

```javascript
/// plugins where imported as postcssPresetEnv and nested, respectively
                   {
                        loader: "postcss-loader",
                        options: {
                            plugins: [
                                postcssPresetEnv({
                                    importFrom: ["./src/variables.css"],
                                }),
                                nested(),
                            ],
                        },
                    },
///
```

Looking at the documentation, I understand that, instead, my config file should look like this:

```javascript
/// 
                   {
                        loader: "postcss-loader",
                        options: {
                            postcssOptions: {
                                plugins: [
                                    "postcss-preset-env",
                                    {
                                        importFrom: ["./src/variables.css"]
                                    },
                                    "postcss-nested"
                                ]
                            }
                        }
                    }
///
```

However, this generates the following error: `Loading PostCSS "importFrom" plugin failed: Cannot find module 'importFrom'`

I had to instead do the following:

```javascript
/// note the extra set of brackets [ ] around the plugin and its options
                       {
                        loader: "postcss-loader",
                        options: {
                            postcssOptions: {
                                plugins: [
                                    [
                                        "postcss-preset-env",
                                        {
                                            importFrom: ["./src/variables.css"]
                                        }
                                    ]
                                ]
                            }
                        }
                    }
///
```

The nested plugin never worked, with any syntax permutation that I tried, and that may be unrelated to this issue  as it also had a major version update.

Are the docs wrong, is the loader working differently from what's expected?

**Comment by evilebottnawi** [comment]
Not sure why it is not work because you ignore the issue template

**Comment by evilebottnawi** [comment]
```js
{
  postcssOptions: {
    plugins: [
      [ "postcss-preset-env", { importFrom: ["./src/variables.css"] }],
      "postcss-nested"
    ]
  }
}
```

**Comment by pablobarria** [comment]
I did not ignore the issue template. I tried to follow two different ones:

![image](https://user-images.githubusercontent.com/5231015/93610768-e24e7700-f9a3-11ea-8eaa-24406885ca42.png)

 but neither of them fit this particular issue, because **I don't know if this is a bug or a problem with the documentation**, and there is no template that targets that particular case. 

If you're telling me that the code as presented in your last comment is the correct version, then this is a documentation issue and the Readme needs to be fixed before it is closed.
